vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/steps/KeyboardSteps.java

```diff
@@ -106,7 +107,20 @@ public void typeTextInFieldAndKeepKeyboard(String text, Locator locator)
     public void typeTextInField(String text, Locator locator)
     {
         baseValidations.assertElementExists(ELEMENT_TO_TYPE_TEXT, locator)
-                .ifPresent(e -> keyboardActions.typeTextAndHide(e, text));
+                .ifPresent(e -> {
+                    keyboardActions.typeText(e, text);
+                    keyboardActions.hideKeyboard(e);
+                });
+    }
+
+    /**
+     * Clear a field located by the <b>locator</b>
+     * @param locator locator to find a field
+     */
+    @When("I clear field located `$locator` and keep keyboard opened")
```

Comment:
hm I think it should be "open" => https://english.stackexchange.com/questions/141547/opened-vs-open